src/main/java/nl/goodbytes/xmpp/xep0363/SecureUUID.java

```diff
@@ -25,12 +28,12 @@ public static SecureUUID generate() {
     }
     
     public static SecureUUID fromString(String id) {
-        return new SecureUUID(id);
+    	return new SecureUUID(id);
```

Comment:
Weird whitespace indent?

Reply:
ok. Changed \t to four spaces. fad8c4fd8ee402245b29122752b31e556c5ad96a